src/linter.ts

```diff
@@ -9,10 +9,12 @@ export interface LinterError {
 }
 
 export default class Linter {
+  private workspaceFolder: any;
   private codeDocument: vscode.TextDocument;
 
   constructor(document: vscode.TextDocument) {
     this.codeDocument = document;
+    this.workspaceFolder = vscode.workspace.workspaceFolders[0];
```

Comment:
If we want to leverage the `.workspaceFolders` member, we'll need to migrate away from the `vscode` dependency in `package.json` and use `@types/vscode`. (we should probably do this anyway, but right now without, this fails to compile)

A migration guide can be found here: https://code.visualstudio.com/api/working-with-extensions/testing-extension#migrating-from-vscode